core/internal/cltest/cltest.go

```diff
@@ -1648,3 +1648,17 @@ func MustGetStateForKey(t testing.TB, kst keystore.Eth, key ethkey.KeyV2) ethkey
 func NewBulletproofTxManagerORM(t *testing.T, db *sqlx.DB, cfg pg.LogConfig) bulletprooftxmanager.ORM {
 	return bulletprooftxmanager.NewORM(db, logger.TestLogger(t), cfg)
 }
+
+// ClearDBTables deletes all rows from the given tables
+func ClearDBTables(t *testing.T, db *sqlx.DB, tables ...string) {
```

Comment:
```suggestion
func ClearDBTables(t *testing.T, db *sqlx.DB, tables []string) {
```

Reply:
Hmm.. to reverse what @jmank88 suggested?

Reply:
What would be the purpose of not including this argument? It would just do nothing.

Reply:
It makes the callers cleaner because they don't have to do a verbose instantiation of the `[]string{}`. I don't think variadic implicitly means that zero length is valid - but we could document it if that is not clear :shrug: